index.html

```diff
@@ -19,18 +20,19 @@
             <li class="right-item"> <a href="#"><i class="fas fa-envelope-square"></i></a></li>
         </ul>
     </nav>
-    <main class="main-container">
+   </header>
+    <section class="main-container">
         <section class="intro-header">
             <h1>Hey There.
                 I’m Michael</h1>
         </section>
-        <section class="intro-title">
-            <h3>I’am a Software Developer</h3>
-        </section>
-        <section class="intro-detail">
+       <section>
+        <div class="intro-detail">
```

Comment:
There should be only one `section` container that wraps the headline intro, paragraph, and icons. Moreover, The section has the proper background image (decorations are always backgrounded images, not img tags). The headline section has a background image in the original design, but you are using `img` tag to show the image.